src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
Not sure this is enough ....

What if option == 'xxx,count', or 'count,yyy' ?

I think there is a vector somewhere, to do something similar to:

    if (ciP->uriParamOptions['check'] == "true")
    {
       ...
    }

And, if there isn't, we'll have to implement it ...